build-models.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Ensure the OpenAPI Generator JAR file is set up
+mkdir -p ~/bin/openapitools
+OPENAPI_GENERATOR_JAR=~/bin/openapitools/openapi-generator-cli.jar
+if [ ! -f "$OPENAPI_GENERATOR_JAR" ]; then
+    curl -L https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.7.0/openapi-generator-cli-7.7.0.jar -o "$OPENAPI_GENERATOR_JAR"
```

Comment:
Parametrize the URL (i.e., define as variable and move to top) so that it can be easily updated.